src/main/java/com/core/model/dto/transaction/TransactionResponseDto.java

```diff
@@ -0,0 +1,13 @@
+package com.core.model.dto.transaction;
+
+import lombok.Data;
+
+@Data
+public class TransactionResponseDto {
+    private Long id;
+    private String fromAccount;
+    private String toAccount;
+    private String timeStamp;
+    private String amount;
```

Comment:
I understand that this is response and you can do whatever suits you but why `String` for numbers?

Reply:
Sure, I got you. Thanks